src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    private VerifyData verifyData = new VerifyData();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        SimpleDate startDate = new SimpleDate(dateFrom);
+        SimpleDate endDate = new SimpleDate(dateTo);
+        int[] salaries = new int[names.length];
+
+        for (String entry : data) {
+
+            try {
+                verifyData.verification(entry);
+            } catch (FormatDataException e) {
+                System.out.println("[" + entry + "] - bad format");
```

Comment:
Consider handling invalid data more robustly, possibly by logging the error or throwing an exception, rather than just printing a message.